finetuning/gemma/README.md

```diff
@@ -0,0 +1,57 @@
+# Fine-Tuning Gemma 2B on Beam
+
+## Fine-Tuning
+In `finetune.py`, we are using Low-Rank Adaption (LoRA) to fine-tune the Gemma language model using the Open Assistant dataset. The goal is to use this dataset to improve Gemma's ability to engage in helpful conversations, making it more suitable for assistant-like apps. 
+
+### LoRA
+You can read more about LoRA [here](https://arxiv.org/abs/2106.09685). However, let's briefly discuss what exactly it does and why we chose to use it here. At a high level, LoRA introduces a new small set of weights to the model that we will be training. By limiting our training to these additional weights, we can fine-tune the model much quicker. Additionally, since we are not touching the original weights, the model's initial knowledge base should intact. 
+
+### Training our model
+In this example, we are using an [A100-40](https://www.nvidia.com/en-us/data-center/a100/). We are using mixed precision (FP16) to optimize for speed and memory usage. In this example, we are only training for one epoch. In practice, you can probably train longer and continue to see improvemed results. 
+
+No surprise here, but we are getting our compute via Beam. We are using the `function` decorator so that we can run our fine-tuning application as if it were on our local machine. 
+```python
+@function(
+    volumes=[Volume(name="gemma-ft", mount_path=MOUNT_PATH)],
+    image=Image(
+        python_packages=["transformers", "torch", "datasets", "peft", "bitsandbytes"]
+    ),
+    gpu="A100-40",
+    cpu=4,
+)
+```
+One interesting thing to note above is that we are mounting a volume to our container. This volume is where we have uploaded our intial weights from hugging face and our training dataset. It is also where we will store our additional fine-tuned weights. 
+
+We can start our training by running `python finetune.py`.
```

Comment:
show an abbreviated output from the training process running in the shell

Reply:
also show that the trained models are saved in a volume by showing the output of `beam ls gemma-ft`